frame/collective/src/lib.rs

```diff
@@ -1086,7 +1081,7 @@ mod tests {
 	fn motions_basic_environment_works() {
 		new_test_ext().execute_with(|| {
 			assert_eq!(Collective::members(), vec![1, 2, 3]);
-			assert_eq!(Collective::proposals(), Vec::<H256>::new());
+			assert_eq!(*Collective::proposals(), Vec::<H256>::new());
```

Comment:
What's this about? o.O

Reply:
`*` is using `Deref` to deref to a vec I think

Reply:
Correct.